tangelo/linq/tests/test_simulator.py

```diff
@@ -139,6 +139,25 @@ def test_simulate_mixed_state_save_measures(self):
             assert_freq_dict_almost_equal(sim.all_frequencies, reference_all, 8e-2)
             assert_freq_dict_almost_equal(sim.mid_circuit_meas_freqs, reference_mid, 8e-2)
 
+    def test_simulate_mixed_state_desired_state(self):
+        """ Test mid-circuit measurement (mixed-state simulation) for all installed backends.
+        Mixed-states do not have a statevector representation, as they are statistical mixtures of several quantum states.
+        """
+
+        results = dict()
+        exact = {'11': 0.23046888414227926, '10': 0.7695311158577207}
+        for b in installed_simulator:
```

Comment:
Doesn't feel like you implemented anything for QDK unless I'm wrong. If you only implemented this feature for a few specific backends, list them explicitly.

Reply:
Remove QDK from list of targets.